src/task_scl_structural_habitat.py

```diff
@@ -25,36 +25,344 @@ class str_hab(SCLTask):
+            {
+            'category': 'Sparse herbaceous cover (<15%)',
+            'lcClass': 153,
+            'elevation_zone1': 2000,
+            'elevation_zone2': 2000,
+            'elevation_zone3': 2000
+        },
+            {
+            'category': 'Tree cover, flooded, fresh or brakish water',
+            'lcClass': 160,
+            'elevation_zone1': 3350,
+            'elevation_zone2': 3350,
+            'elevation_zone3': 3350
+        },
+            {
+            'category': 'Tree cover, flooded, saline water',
+            'lcClass': 170,
+            'elevation_zone1': 3350,
+            'elevation_zone2': 3350,
+            'elevation_zone3': 3350
+        },
+            {
+            'category': 'Shrub or herbaceous cover, flooded, fresh/saline/brakish water',
+            'lcClass': 180,
+            'elevation_zone1': 2000,
+            'elevation_zone2': 2000,
+            'elevation_zone3': 2000
+        },
+            {
+            'category': 'Urban areas',
+            'lcClass': 190,
+            'elevation_zone1': 0,
+            'elevation_zone2': 0,
+            'elevation_zone3': 0
+        },
+            {
+            'category': 'Bare areas',
+            'lcClass': 200,
+            'elevation_zone1': 0,
+            'elevation_zone2': 0,
+            'elevation_zone3': 0
+        },
+            {
+            'category': 'Consolidated bare areas',
+            'lcClass': 201,
+            'elevation_zone1': 0,
+            'elevation_zone2': 0,
+            'elevation_zone3': 0
+        },
+            {
+            'category': 'Unconsolidated bare areas',
+            'lcClass': 202,
+            'elevation_zone1': 0,
+            'elevation_zone2': 0,
+            'elevation_zone3': 0
+        },
+            {
+            'category': 'Water bodies',
+            'lcClass': 210,
+            'elevation_zone1': 0,
+            'elevation_zone2': 0,
+            'elevation_zone3': 0
+        },
+            {
+            'category': 'Permanent snow and ice',
+            'lcClass': 220,
+            'elevation_zone1': 0,
+            'elevation_zone2': 0,
+            'elevation_zone3': 0
+        },
+        ]
+
+        lcFrom= []
+        zone1= []
+        zone2= []
+        zone3= []
+
+        for i in range(len(lcClassification)):
+            lcFrom.append(lcClassification[i].get('lcClass'))
+            zone1.append(lcClassification[i].get('elevation_zone1'))
+            zone2.append(lcClassification[i].get('elevation_zone2'))
+            zone3.append(lcClassification[i].get('elevation_zone3'))
+
+        str_hab= remap_lc_by_zone(lcFrom, zone1, zone2, zone3, zones_img, self.elevation, self.land_cover)
+        print(str_hab)
         exit()
 
-        # export_path = "scl_poly/{}/scl_polys".format(self.taskdate)
-        #
-        # self.export_fc_ee(scl_polys, export_path)
+# need to do the following:
+    # set up exports (including asset location/naming)
+    # aoi
```

Comment:
See above -- you shouldn't need to worry about aoi unless you're just testing on a small area